setuptools/command/install_lib.py

```diff
@@ -74,10 +74,10 @@ def _gen_exclusion_paths():
         yield '__init__.pyc'
         yield '__init__.pyo'
 
-        if not hasattr(imp, 'get_tag'):
+        if not hasattr(sys, 'implementation'):
             return
 
-        base = os.path.join('__pycache__', '__init__.' + imp.get_tag())
+        base = os.path.join('__pycache__', '__init__.' + sys.implementation.cache_tag)
```

Comment:
The docs say `sys.implementation.cache_tag` may be `None` (in which case you're supposed to disable module caching).  Maybe check for that too and do an early return?

Reply:
this isn't a regression, `imp.get_tag()` has identical semantics and is implemented as just `return sys.implementation.cache_tag`: https://github.com/python/cpython/blob/9b5ce62cac27fec9dea473865d79c2c654312957/Lib/imp.py#L67-L69